app/models/room_guest.rb

```diff
@@ -0,0 +1,4 @@
+class RoomGuest <ApplicationRecord
+  belongs_to :room
+  belongs_to :guest
+end
```

Comment:
All of your associations across all of your models are correct and set up properly.